packages/web-components/src/components/leadspace-block/__stories__/leadspace-block.stories.ts

```diff
@@ -38,7 +38,9 @@ const image = html`
 
 const linkList = html`
   <dds-link-list type="end">
-    <dds-link-list-heading>Featured products</dds-link-list-heading>
+    <dds-link-list-heading aria-level="3"
```

Comment:
@annawen1 I'm wondering if it would be better to dynamically setting this within the `leadspace-block` component instead of in the story? I'm assuming the issue will remain if an adopter uses the same layout from our story, so maybe we could loop through the slotted elements of the `leadspace-block`, count how many `aria-heading`s we have, and make sure any wrong levels are adjusted. Or if the `link-list-heading` the only affected component, we could make it always be `aria-level=3` when used within `leadspace-block`.

Reply:
Ignore all this if this bug is a Storybook only bug though!

Reply:
Hmmm my concern is adding all that logic can make the component less flexible, but if `link-list` will always be part of `leadspace-block` then setting the `aria-level` within `leadspace-block` makes sense

Reply:
According to the specs, the `link-list` is required so I'll adjust the `aria-level` for `link-list` within the component!